mongo_orchestration/sharded_clusters.py

```diff
@@ -266,6 +266,12 @@ def router(self):
 
     def router_add(self, params):
         """add new router (mongos) into existing configuration"""
+        # featureFlagLoadBalancer was added in 5.0.7 (SERVER-60679) and
+        # removed in 6.1.0 (SERVER-64205).
+        if (5, 0, 7) <= self.mongos_version[:3] <= (6, 0, 99):
```

Comment:
Hopefully we'll never get to 99 patch releases for 6.0.

Reply:
Fixed :)